docs/large_cohort_pipeline_readme.md

```diff
@@ -0,0 +1,235 @@
+<br><ins>Configurable inputs:</ins>
+	* [workflow].skip_qc
+	* [workflow].check_intermediates
+	* [workflow].[sequencing_type]
+
+3. GvcfHappy: This step runs [Hap.py](https://github.com/Illumina/hap.py/blob/master/doc/happy.md "") validation/concordance stats to validate a GVCF for samples where a truth callset is available. The expected output is a MultiQC `*.summary.csv` file. 
+<br><ins>Configurable inputs:</ins>
+	* [workflow].[sequencing_type]
+	* get_config().get(‘validation’, {}).get()sample_map) - ==where does this come from?==
+
+4. GvcfMultiQC: This step runs MultiQC to summarise all GVCF QC output, with an HTML and corresponding JSON file produced.
+<ins>Configurable inputs:</ins>
+	* [workflow].skip_qc
+	* [workflow].dry_run
+	* get_config().get('qc_thresholds') - ==where does this come from?==
+### FrequenciesStage
+1. Genotype: This step uses the `genotype` job to run the HaplotypeCaller, which genotypes individual samples (i.e. CRAM -> GVCF) and produces a GVCF + corresponding TBI index. 
+<br><ins>Configurable inputs:</ins>
+	* [workflow].check_intermediates
+	* [workflow].intervals_path
+	* [workflow].[sequencing_type]
+
+2. Combiner: This step runs the Hail [VDS combiner](https://hail.is/docs/0.2/vds/hail.vds.combiner.VariantDatasetCombiner.html#hail.vds.combiner.VariantDatasetCombiner ""), which produces a VariantDataset (VDS) by combining any number of GVCF and/or VDS files. For more information, see Hail’s documenatation on [Variant Datasets](https://hail.is/docs/0.2/vds/index.html ""). The output of this step is written to `gs://cpg-{dataset}-{access-level}/vds/v0-1.vds`.
+<br><ins>Configurable inputs:</ins>
+	* [workflow].vds_version
+	* [hail].dataproc.combiner_autoscaling_policy
+	* get_config().get('combiner', {})
+	* [workflow].[sequencing_type]
+	* [workflow].check_inputs
+	* [workflow].skip_samples_with_missing_input
+
+3. SampleQC: This step takes an input VDS table and performs the following steps: 1. removes centromeres and telomeres 2. filters to autosomes and 3. computes sample quality metrics about the VDS (==what specifically are these? Can’t find in the `vds.sample_qc` docs==). Sex is then imputed on the VDS and the following soft filters added: samples with ambiguous sex assignments, low-coverage samples, and extreme raw bi-allelic sample QC outliers. Sample-QC based filters are calculated according to the thresholds specified in the `large_cohort.sample_qc_cutoffs` section of the `configs/defaults/large_cohort.toml`. The output of this step is written to `gs://cpg-{dataset}-{access-level}/large-cohort/v0-1/sample_qc.ht`. 
```

Comment:
@vladsavelyev @lgruen would you happen to have any extra info of what sample quality metrics are used here? I couldn't find it in the docs

Reply:
It's configured through toml as well: https://github.com/populationgenomics/production-pipelines/blob/main/configs/defaults/large_cohort.toml#L30-L36

The thresholds are borrowed from gnomAD QC, e.g. https://macarthurlab.org/2017/02/27/the-genome-aggregation-database-gnomad/

Reply:
Thanks @vladsavelyev . I can't find all the QC metrics in that link. Is it possible to give another source where you found those, and what gnomAD version it corresponds to?